packages/connect-explorer/README.md

```diff
@@ -1,8 +1,14 @@
-# TrezorConnect explorer
+# @trezor/connect-explorer
 
 This package serves as:
 
 -   an interactive documentation for [@trezor/connect](../connect) package
--   a referential implementation of @trezor/connect-web in "popup mode"
+-   a referential implementation of [@trezor/connect-web](../connect-web) in "popup mode"
 
-`yarn dev` - runs dev server on port 8082
+To run the explorer locally:
+
+`yarn workspace @trezor/connect-explorer dev` - runs dev server on port 8088
+
+When `connect-exporer` is run locally, WebUSB is disabled by Chrome because [@trezor/connect-popup](../connect-popup) runs on a different domain.
```

Comment:
webusb is disabled in popup because we cannot call `navigator.usb.requestDevice` between cross-site elements anymore (changed by chrome v72) i dont know if it's worth to mention that, it's more like a bug or lack of feature from our side which hopefully will be fixed one day :)